.idea/encodings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
you shouldn't have your pycharm settings in the git repo. add this folder to `.gitignore`